demisto_sdk/commands/common/update_id_set.py

```diff
@@ -397,7 +397,7 @@ def get_test_playbooks_paths():
     return test_playbook_files
 
 
-def re_create_id_set():
+def re_create_id_set(output='./Tests/id_set.json'):
```

Comment:
Is it possible to use the id set result without creating `id_set.json` file? Just to return it as dict?

Reply:
assuming it's already there?

Reply:
if you pass `None` as output it will not create the file.

Reply:
👍